src/main/java/org/fundacionjala/coding/Yuri/Movie/Customer.java

```diff
@@ -0,0 +1,88 @@
+package org.fundacionjala.coding.Yuri.Movie;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+class Customer {
+    private static final double CONSTANT_DAY_RENTED = 1.5;
+    private static final double CONSTANT_REGULAR_MOVIE = 2;
+    private static final double CONSTANT_CHILDREN_MOVIE = 3;
+    private static final double DAY_RENTED_NEW_RELEASE = 3;
+    private static final double CONSTANT_NEW_RELEASE = 0.0;
+    private String name;
+    private Vector rentals = new Vector();
+
+    /**
+     * this is the constructor method.
+     * @param name the name of the customer.
+     */
+    public Customer(String name) {
+        this.name = name;
+    }
+
+    public void addRental(Rental arg) {
+        rentals.addElement(arg);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String statement() {
```

Comment:
This method is still doing 3 things. it is calculating the totalAmount, totalFrequentRenterPoints and is printing.
Please split it.